abstract_specification.md

```diff
@@ -422,8 +422,8 @@ Summary of operations:
 | get ndims -> int           | Return number of dimensions.                                                   |
 | get schema -> Arrow.Schema | Return data schema, in the form of an Arrow Schema.                            |
 | get is_sparse -> False     | Return the constant False.                                                     |
-| read                       | Read a slice of data from the SOMADenseNdArray.                                |
-| write                      | Write a slice of data to the SOMADenseNdArray.                                 |
+| read                       | Read a subarray from the SOMADenseNdArray.                                     |
```

Comment:
nit: trailing spaces?

Reply:
trailing spaces are correct formatting for markdown in this repo (using the default `prettier` conventions).  This makes the table square, and easier to read in source form.

Reply:
derp sorry @bkmartinjr :)

Reply:
ps. I'm not averse to changing the formatting rules, as long as they are 100% automated and I can just run a formatter each time I hit save :-)